pkg/plugins/plugins.go

```diff
@@ -269,6 +269,15 @@ func (scanner *PluginScanner) loadPluginJson(pluginJsonFilePath string) error {
 		return errors.New("did not find type or id properties in plugin.json")
 	}
 
+	if pluginCommon.Id == "gel" {
+		isEnabled := scanner.cfg.FeatureToggles["expressions"]
+		scanner.log.Debug("Plugin is dependent on a feature flag, checking if enabled", "pluginID", pluginCommon.Id,
+			"featureFlag", "expressions", "enabled", isEnabled)
+		if !isEnabled {
+			return fmt.Errorf("plugin %q is disabled since the expressions feature is not toggled", pluginCommon.Id)
```

Comment:
Since the plugin is bundled, does it make sense to error if not enabled? Seems okay to be silent on not loading without the feature flag enabled.

Reply:
@kylebrandt Yeah, I guess that makes sense. Pushing an edit.

Reply:
@kylebrandt On second thought, I remember now why I have these errors, it's in order to be able to test. The errors are actually silent to the user (just recorded on an object), so no there's no problem.